spec/actions_specs/unlock_keychain_spec.rb

```diff
@@ -0,0 +1,21 @@
+describe Fastlane do
+  describe Fastlane::FastFile do
+    describe "Unlock keychain Integration" do
+
+      it "works with path and password and existing keychain" do
+        keychain_path = Tempfile.new('foo').path
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.